.changeset/tiny-walls-tan.md

```diff
@@ -0,0 +1,8 @@
+---
+"@nomicfoundation/hardhat-ethers-chai-matchers": patch
```

Comment:
I think this is missing a few packages that got their network hook handlers updated.

Reply:
I updated the changesets here: https://github.com/NomicFoundation/hardhat/pull/6595/commits/74748831e478922ad8136f498e7589ec104b5365